components/03-modules/dashboard-items/_dashboard-items.scss

```diff
@@ -0,0 +1,27 @@
+$dashboard-items-padding: 15px !default;
+$dashboard-items-margin : 15px !default;
+
+.dashboard-items {
+    display: flex;
+    flex-direction: column;
+    padding: $dashboard-items-padding;
```

Comment:
I'm just curious - why you use flex (column) in here?

Reply:
yes, it's true I can delete this whole flex. Checked, and I think everything is still working properly.

Reply:
👍